andela/titan-dev-ah-frontend
# #166059552 Home Page

#### What does this PR do?

- It designs the homepage for the application
#### Description of Task to be completed?

- Design the homepage, fetch featured, popular articles. 
- Handle the fetch actions.

#### How should this be manually tested?
- After cloning the repository, check out `ft-create-the-homepage-166059552` branch
- Run `yarn` to install the project dependencies
- Run `yarn start` to start the application then, you navigate to any of the articles

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #166059552
